### PR TITLE
Fix unable to find dependent proto issue in gRPC cli

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose
Related to https://github.com/ballerina-platform/ballerina-standard-library/issues/2711
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [x] Added tests

### CODE REVIEW

**Comment by BuddhiWathsala** [comment]
I tried my actual requirement with this fix, but still, those errors are there. Could you try to generate stubs for the following proto definition.

https://github.com/envoyproxy/java-control-plane/blob/v0.1.30/api/src/main/proto/envoy/service/discovery/v3/ads.proto

**Comment by MadhukaHarith92** [comment]
> I tried my actual requirement with this fix, but still, those errors are there. Could you try to generate stubs for the following proto definition?.
> 
> https://github.com/envoyproxy/java-control-plane/blob/v0.1.30/api/src/main/proto/envoy/service/discovery/v3/ads.proto

Will check with this example.

**Comment by BuddhiWathsala** [comment]
Try to generate using the attached zip here. I was able to narrow it down to a single error by adding missing protos here.

Now, the error seems like.
```sh
Continuing with the existing protoc executor file at /var/folders/jq/ps0xmq910yv6mqfpqk5w6zd00000gp/T/protoc-3.19.1-osx-x86_64.exe
Successfully extracted the library files.
Feb 22, 2022 10:14:56 PM io.ballerina.stdlib.grpc.protobuf.cmd.GrpcCmd generateBalFile
SEVERE: An error occurred when generating the dependent proto descriptor. 
google/protobuf/descriptor.proto:43:8: Option "go_package" was already set.
io.ballerina.stdlib.grpc.protobuf.exception.CodeGeneratorException: 
google/protobuf/descriptor.proto:43:8: Option "go_package" was already set.
        at io.ballerina.stdlib.grpc.protobuf.utils.BalFileGenerationUtils.generateDescriptor(BalFileGenerationUtils.java:81)
        at io.ballerina.stdlib.grpc.protobuf.cmd.DescriptorsGenerator.generateDependentDescriptor(DescriptorsGenerator.java:92)
        at io.ballerina.stdlib.grpc.protobuf.cmd.DescriptorsGenerator.generateDependentDescriptor(DescriptorsGenerator.java:98)
        at io.ballerina.stdlib.grpc.protobuf.cmd.DescriptorsGenerator.generateDependentDescriptor(DescriptorsGenerator.java:98)
        at io.ballerina.stdlib.grpc.protobuf.cmd.DescriptorsGenerator.generateDependentDescriptor(DescriptorsGenerator.java:98)
        at io.ballerina.stdlib.grpc.protobuf.cmd.DescriptorsGenerator.generateDependentDescriptor(DescriptorsGenerator.java:98)
        at io.ballerina.stdlib.grpc.protobuf.cmd.DescriptorsGenerator.generateDependentDescriptor(DescriptorsGenerator.java:98)
        at io.ballerina.stdlib.grpc.protobuf.cmd.GrpcCmd.generateBalFile(GrpcCmd.java:271)
        at io.ballerina.stdlib.grpc.protobuf.cmd.GrpcCmd.execute(GrpcCmd.java:156)
        at java.base/java.util.Optional.ifPresent(Optional.java:183)
        at io.ballerina.cli.launcher.Main.main(Main.java:51)
```

[proto.zip](https://github.com/ballerina-platform/module-ballerina-grpc/files/8118642/proto.zip)

File: tooling-tests/src/test/resources/test-src/proto-files/external-imports/myproj/foo/bar/child.proto [hunk 1]
```diff
@@ -18,13 +18,16 @@
...
 
+import "google/protobuf/wrappers.proto";
 import "parent.proto";
 
 service ChildTest {
     rpc callChild1 (ParentMessage) returns (ChildMessage);
     rpc callChild2 (stream ParentMessage) returns (ChildMessage);
     rpc callChild3 (ParentMessage) returns (stream ParentMessage);
     rpc callChild4 (stream ParentMessage) returns (stream ParentMessage);
+
```
Comment (BuddhiWathsala) [inline tooling-tests/src/test/resources/test-src/proto-files/external-imports/myproj/foo/bar/child.proto:29]:
```suggestion
```
Removed unnecessary space.

**Review by BuddhiWathsala — APPROVED** [review]
LGTM

**Comment by BuddhiWathsala** [comment]
> Try to generate using the attached zip here. I was able to narrow it down to a single error by adding missing protos here.
> 
> Now, the error seems like.
> 
> ```shell
> Continuing with the existing protoc executor file at /var/folders/jq/ps0xmq910yv6mqfpqk5w6zd00000gp/T/protoc-3.19.1-osx-x86_64.exe
> Successfully extracted the library files.
> Feb 22, 2022 10:14:56 PM io.ballerina.stdlib.grpc.protobuf.cmd.GrpcCmd generateBalFile
> SEVERE: An error occurred when generating the dependent proto descriptor. 
> google/protobuf/descriptor.proto:43:8: Option "go_package" was already set.
> io.ballerina.stdlib.grpc.protobuf.exception.CodeGeneratorException: 
> google/protobuf/descriptor.proto:43:8: Option "go_package" was already set.
>         at io.ballerina.stdlib.grpc.protobuf.utils.BalFileGenerationUtils.generateDescriptor(BalFileGenerationUtils.java:81)
>         at io.ballerina.stdlib.grpc.protobuf.cmd.DescriptorsGenerator.generateDependentDescriptor(DescriptorsGenerator.java:92)
>         at io.ballerina.stdlib.grpc.protobuf.cmd.DescriptorsGenerator.generateDependentDescriptor(DescriptorsGenerator.java:98)
>         at io.ballerina.stdlib.grpc.protobuf.cmd.DescriptorsGenerator.generateDependentDescriptor(DescriptorsGenerator.java:98)
>         at io.ballerina.stdlib.grpc.protobuf.cmd.DescriptorsGenerator.generateDependentDescriptor(DescriptorsGenerator.java:98)
>         at io.ballerina.stdlib.grpc.protobuf.cmd.DescriptorsGenerator.generateDependentDescriptor(DescriptorsGenerator.java:98)
>         at io.ballerina.stdlib.grpc.protobuf.cmd.DescriptorsGenerator.generateDependentDescriptor(DescriptorsGenerator.java:98)
>         at io.ballerina.stdlib.grpc.protobuf.cmd.GrpcCmd.generateBalFile(GrpcCmd.java:271)
>         at io.ballerina.stdlib.grpc.protobuf.cmd.GrpcCmd.execute(GrpcCmd.java:156)
>         at java.base/java.util.Optional.ifPresent(Optional.java:183)
>         at io.ballerina.cli.launcher.Main.main(Main.java:51)
> ```
> 
> [proto.zip](https://github.com/ballerina-platform/module-ballerina-grpc/files/8118642/proto.zip)

Let's address this issue separately.

**Pull request merged**